test/Board/board.test.js

```diff
@@ -1,54 +1,53 @@
 import { Board } from '../../lib/Board/board'
+import { Token } from '../../lib/Token/token'
 
 describe('Board', () => {
     let board;
 
-    
     beforeEach(() => {
         board = new Board();
     });
 
-    describe('getCells()', () => {
-        test('will return a structure storing each individual cell in the board', () => {
-            const cellStructure = board.getCells()
-            expect(cellStructure).toBe(board.cells)
-        });    
-    });
-
-    describe('formatCells()', () => {
-        test('will format a cell without a column separator', () => {
-            const formattedCellWithSeparator = board.formatCells(board.cells[0][2], 2)
-            expect(formattedCellWithSeparator).toBe(` `)
+    describe('toString()', () => {
+        test('will display 1 row when the board is instantiated with only 1 row', () => {
+            board = new Board(1, 3);
+            const row = board.toString();
+            expect(row).toBe(`   |    |  `);
         });
 
-        test('will format a cell with a column separator', () => {
-            const formattedCellNoSeparator = board.formatCells(board.cells[0][0], 0)
-            expect(formattedCellNoSeparator).toBe(`   |`)
+        test('will display a standard, user-friendly 3x3 Tic-Tac-Toe board', () => {
+            const printedBoard = board.toString();
+            const rowOutput = `   |    |  `;
+            expect(printedBoard).toBe(`${rowOutput}\n${board.rowSeparator}\n${rowOutput}\n${board.rowSeparator}\n${rowOutput}`);
         });
     });
 
-    describe('toString()', () => {
-        test('will print 1 row when the board is instantiated with only 1 row', () => {
-            const board = new Board(1, 3)
-            const row = board.toString()
-            expect(row).toBe(`   |    |  `)
-        });
+    describe('getCells()', () => {
+        test('will return an array data structure used to store each row and column in the board', () => {
+            const cellStructure = board.getCells();
+            expect(cellStructure).toBe(board.cells);
+        });    
+    });
 
-        test('will print a standard, user-friendly 3x3 Tic-Tac-Toe board', () => {
-            const printedBoard = board.toString()
-            const rowOutput = `   |    |  `
-            expect(printedBoard).toBe(`${rowOutput}\n${board.rowSeparator}\n${rowOutput}\n${board.rowSeparator}\n${rowOutput}`)
+    describe('getRow()', () => {
+        test('will return the correlated row number for a valid position', () => {
```

Comment:
@DeKaylaPhillips Can you explain what this function is doing? Why does `board.getRow(#)` return a number?